cfgov/housing_counselor/results_archiver.py

```diff
@@ -0,0 +1,21 @@
+import json
+from zipfile import ZIP_DEFLATED, ZipFile
+
+
+json_file_path = './HUD_approved_housing_counselors.json'
+
+
+# By request from stakeholders in the Office of Innovation, save a
+# list of the full results we receive from the HUD API, in case of
+# future enforcement actions.
+
+# As an MVP, we're saving the complete JSON response, zipped. As a future
+# enhancement, we could consider improving the format of the data,
+# e.g. as CSV.
+def save_list(counselors, destination_path):
+    with open(json_file_path, 'w') as f:
+        f.write(json.dumps(counselors))
+
+    zip = ZipFile(destination_path, 'w', ZIP_DEFLATED, allowZip64=True)
+    zip.write(json_file_path)
+    zip.close
```

Comment:
```suggestion
    zip.close()
```

This is missing the parenthesis, so isn't actually calling the `close` method. Another way of doing this would be to use a context manager like

```py
with ZipFile(...) as zf:
    zf.write(json_file_path)
```

Additionally, [`zip`](https://docs.python.org/3/library/functions.html#zip) is a built-in method in Python so it might be better to use a different variable name.

Reply:
Thanks for that. I missed that part of the ZipFile documentation. Updated.